app/js/postEditFunction.js

```diff
@@ -0,0 +1,40 @@
+/*
+    Creates an object of the user from the data in the input fields
+    Should be called within the submit event listener
+
```

Comment:
each line should start with a * and the @param should be followed by the name of the parameter.

Reply:
resolved